consensus-types/types/body.go

```diff
@@ -96,7 +96,7 @@ type BeaconBlockBody struct {
 	blsToExecutionChanges []*BlsToExecutionChange
 	// BlobKzgCommitments is the list of KZG commitments for the EIP-4844 blobs.
 	BlobKzgCommitments []eip4844.KZGCommitment
-	// executionRequests is introduced in electra. We keep this private so that it must go through Getter/Setter
+	// executionRequests is introduced in Electra. We keep this private so that it must go through Getter/Setter
```

Comment:
just noting that this is an anti-pattern from the rest of consensus types. fields are kept private iff they are unused and satisfy `EnforceUnused()`. Unsure if we should keep this field public or allow private to imply that its not _always_ used.